### PR TITLE
Potential new code of conduct 

### PR DESCRIPTION
**Pull request by russdaygh**
Replaces the existing code of conduct with the ant-harassment policy from the Geek Feminism wiki: https://geekfeminism.fandom.com/wiki/Conference_anti-harassment/Policy

This also includes an internal policy and guidance for staff/volunteers.

### CODE REVIEW

File: src/routes/code-of-conduct/+page.svelte [hunk 1]
```diff
@@ -11,46 +11,67 @@
...
-			Harassment includes offensive verbal comments related to gender, age, sexual orientation,
-			disability, physical appearance, body size, race, religion, sexual images in public spaces,
-			deliberate intimidation, stalking, following, harassing photography or recording, sustained
-			disruption of talks or other events, inappropriate physical contact, and unwelcome sexual
-			attention. Participants asked to stop any harassing behavior are expected to comply
-			immediately.
+			Harassment includes, but is not limited to:
 		</p>
+		<ul>
+			<li>Verbal comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, or religion.</li>
```
Comment (dynamictulip) [inline src/routes/code-of-conduct/+page.svelte:20]: Given the current situation in Gaza I think we should have another point that explicitly states our position on things that could be perceived as antisemitic as most definitions of antisemitism are tied to the existence of the state of Israel. One for discussion?

**Review by dynamictulip — COMMENTED** [review]
This is awesome and way better than what we had!
Reading through I came up with a few questions, we can discuss these further at some point if you like?
There's also a couple of small formatting things and I noticed a few americanisms creeping in from the template ("offense" vs "offence", "behavior" vs "behaviour" etc) which we might want to correct

File: src/routes/code-of-conduct/+page.svelte [hunk 2]
```diff
@@ -11,46 +11,67 @@
...
+			<li>Sexual images in public spaces</li>
+			<li>Deliberate intimidation, stalking, or following </li>
+			<li>Harassing photography or recording</li>
+			<li>Sustained disruption of talks or other events</li>
+			<li>Inappropriate physical contact</li>
+			<li>Unwelcome sexual attention</li>
+			<li>Advocating for, or encouraging, any of the above behaviour</li>
+		</ul>
```
[the remark this answers is not in the document]
Comment (russdaygh) [reply src/routes/code-of-conduct/+page.svelte:29]: I've set `<li>` to the same font size as `<p>` now. It's not reflecting in the page previews though which is odd. I find that they are quite flakey and seem to produce somewhat random results.

File: src/routes/code-of-conduct/+page.svelte [hunk 3]
```diff
@@ -11,46 +11,67 @@
...
+			<li>Contacting a staff member, identified by DDD South West 2024 t-shirts or lanyards.</li>
+		</ul>
+		<p>
+			When taking a personal report, our staff will ensure you are safe and cannot be overheard. They may involve other event staff to ensure your report is managed properly. Once safe, we'll ask you to tell us about what happened. This can be upsetting, but we'll handle it as respectfully as possible, and you can bring someone to support you. You won't be asked to confront anyone and we won't tell anyone who you are.
+		
+			Our team will be happy to help you contact hotel/venue security, local law enforcement, local support services, provide escorts, or otherwise assist you to feel safe for the duration of the event. We value your attendance.
 		</p>
+		<h3>Credit</h3>
+        <p>
+            This anti-harassment policy is based on the example policy from the Geek Feminism wiki, created by the Ada Initiative and other volunteers.
```
[the remark this answers is not in the document]
Comment (russdaygh) [reply src/routes/code-of-conduct/+page.svelte:85]: I've added a link to the policy now in both documents.

**Comment by russdaygh** [comment]
> This is awesome and way better than what we had! Reading through I came up with a few questions, we can discuss these further at some point if you like? There's also a couple of small formatting things and I noticed a few americanisms creeping in from the template ("offense" vs "offence", "behavior" vs "behaviour" etc) which we might want to correct

I've addressed the Americanisms. Do you have questions in addition to the ones on this PR or are they all here?

File: src/routes/code-of-conduct/internal/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        <h2>Expulsion</h2>
+        <p>
+            A participant may be expelled by the decision of any of the above listed entities for whatever reasons they deem sufficient. However, here are some general guidelines for when a participant should be expelled:
+        </p>
+        <ul>
+            <li>A second offense resulting in a warning from staff</li>
+            <li>Continuing to harass after any "No" or "Stop" instruction</li>
+            <li>A pattern of harassing behavior, with or without warnings</li>
+            <li>A single serious offense (e.g., punching or groping someone)</li>
+            <li>A single obviously intentional offense (e.g., taking up-skirt photos)</li>
```
[the remark this answers is not in the document]
Comment (russdaygh) [reply src/routes/code-of-conduct/internal/+page.svelte]: Thanks, I've updated the policy to reflect this.

**Review by dynamictulip — APPROVED** [review]

**Pull request merged**